src/legacy/core_plugins/kibana/public/dashboard/listing/dashboard_listing.js

```diff
@@ -38,6 +43,124 @@ export class DashboardListing extends React.Component {
     super(props);
   }
 
+  getCurrentSpace() {
```

Comment:
Spaces are also an X-Pack (Elastic License) concept, so we can't reference them from within OSS (Apache 2.0) code.